src/core/toxid.h

```diff
@@ -41,9 +41,9 @@ class ToxId
 public:
     ToxId(); ///< The default constructor. Creates an empty Tox ID.
     ToxId(const ToxId& other); ///< The copy constructor.
-    ToxId(const QString& id); ///< Create a Tox ID from QString.
-                              /// If the given id is not a valid Tox ID, then:
-                              /// publicKey == id and noSpam == "" == checkSum.
+    explicit ToxId(const QString& id); ///< Create a Tox ID from QString.
+                                       /// If the given id is not a valid Tox ID, then:
+                                       /// publicKey == id and noSpam == "" == checkSum.
```

Comment:
Nah, it shouldn't. Should be at the right place in cpp file. Up to now qTox gives a sh** about docs, but if you want to help improve that situation, please move that to cpp. Not only, but especially in this case, Qt shows how it is done right! They will even reject a commit, if undocumented. We didn't reach a point yet where we need to be this restrictive, but I personally strive for the best. :) Btw. didn't you want to put it in `/**…*/` blocks?